erpnext/erpnext_integrations/doctype/affirm_settings/affirm_settings.py

```diff
@@ -234,38 +252,37 @@ def authorize_payment(affirm_data, reference_doctype, reference_docname, integra
 		frappe.msgprint(affirm_data.get("message"))
 		redirect_url = "/cart"
 	else:
-		payment_successful(affirm_data, reference_doctype, reference_docname, integration_request)
+		payment_successful(affirm_data, sales_order)
+		integration_request.update_status(affirm_data, "Completed")
 		redirect_url = '/integrations/payment-success'
-
+	
```

Comment:
unnecessary tab

Reply:
These tabs are absolutely fine on my vscode, why is it showing up like this on github?